src/js/pyodide.js

```diff
@@ -5,6 +5,7 @@
 import { Module } from "./module";
 import { loadScript, initializePackageIndex } from "./load-pyodide";
 import { PUBLIC_API, makePublicAPI } from "./api";
+import {} from "./pyproxy";
```

Comment:
It this just to check that `./pyproxy.js` exists or does it do something else?

Reply:
I am importing `pyproxy.js` for its side effects (namely adding a bunch of stuff to `Module`). [There is a special syntax for importing things for their side effects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#import_a_module_for_its_side_effects_only) so we should probably use that:
```suggestion
`import "./pyproxy";`
```